src/Protocol/Feed.php

```diff
@@ -354,7 +354,7 @@ public static function import($xml, array $importer = [], array $contact = [])
 
 			$item["plink"] = DI::httpRequest()->finalUrl($item["plink"]);
 
-			$item["parent-uri"] = $item["uri"];
+			$item["thr-parent"] = $item["uri"];
```

Comment:
Is this really needed?

Reply:
No, this was from before I started to remove self-assigned `parent-uri` and `thr-parent` before passing to `Item::insert`.